react/src/lib/EventOverlay/index.js

```diff
@@ -224,69 +212,67 @@ class EventOverlay extends React.Component {
       top: rect.top - parentRect.top,
       left: rect.left - parentRect.left,
       width: node.offsetWidth,
-      height: node.offsetHeight
+      height: node.offsetHeight,
     };
 
     anchorPosition.right =
       (rect.right || anchorPosition.left + anchorPosition.width) - parentRect.left;
     anchorPosition.bottom =
       (rect.bottom || anchorPosition.top + anchorPosition.height) - parentRect.top;
-    anchorPosition.middle =
-      anchorPosition.left + (anchorPosition.right - anchorPosition.left) / 2;
-    anchorPosition.center =
-      anchorPosition.top + (anchorPosition.bottom - anchorPosition.top) / 2;
+    anchorPosition.middle = anchorPosition.left + (anchorPosition.right - anchorPosition.left) / 2;
```

Comment:
lots of formatting changes. might be an issue in general while working with Momentum - i've been scrolling for awhile and still haven't gotten to the change